client/jetpack-cloud/sections/onboarding-tours/add-new-site-tour-step-2/index.tsx

```diff
@@ -0,0 +1,48 @@
+import { useTranslate } from 'i18n-calypso';
+import { useContext } from 'react';
+import GuidedTour from 'calypso/jetpack-cloud/components/guided-tour';
+import { useSelector } from 'calypso/state';
+import { items } from 'calypso/state/notices/reducer';
+import { getPreference } from 'calypso/state/preferences/selectors';
+import SitesOverviewContext from '../../agency-dashboard/sites-overview/context';
+
+export default function AddNewSiteTourStep2() {
+	const translate = useTranslate();
+	const { mostRecentConnectedSite } = useContext( SitesOverviewContext );
+	const hasAddNewSiteTourPreference = useSelector( ( state ) =>
+		getPreference( state, 'jetpack-cloud-site-dashboard-add-new-site-tour-step-1' )
+	);
+	const shouldRenderAddSiteTourStep2 = hasAddNewSiteTourPreference && mostRecentConnectedSite;
```

Comment:
Could we add a comment here to clarify why we are doing this?

Reply:
Added in https://github.com/Automattic/wp-calypso/pull/85488/commits/a1ed19bd4e770806c936521279644a2c66abf566